universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -379,6 +375,38 @@ private void putParentArray(Path path) {
     return readString(path);
   }
 
+  /**
+   * Delete the ENTIRE repeated entity for each entity specified by the path to an array element and
+   * list of indices.
```

Comment:
nit: please add comment explaining return value (under what cases is it true/false)

Reply:
done